src/MicroflowTimer/widget/MicroflowTimer.js

```diff
@@ -197,6 +207,30 @@ define([
             }
         },
 
+        _executeNanoFlow: function() {
+            logger.debug(this.id + "._executeNanoFlow");
+            if (!this._contextObj) {
+                return;
+            }
+
+            if (this.nanoflow.nanoflow) {
```

Comment:
check for `mxcontext` as well ` if (this.nanoflow.nanoflow && this.mxcontext) `

Reply:
done